public/src/client/topic/events.js

```diff
@@ -78,6 +80,16 @@ define('forum/topic/events', [
         reputationElements.html(data.user.reputation).attr('data-reputation', data.user.reputation);
     }
 
```

Comment:
specifies the function updateEndorsed changes the endorsement status to be not hidden and creates html changes that the user can use to identify a post is endorsed